pluma/pluma-document.c

```diff
@@ -704,17 +709,35 @@ set_language (PlumaDocument     *doc,
 	if (old_lang == lang)
 		return;
 
-	bom_langs = "asp,dtl,docbook,html,mxml,mallard,markdown,mediawiki,php,tera,xml,xslt";
+	new_lang_id = gtk_source_language_get_id (lang);
+	if (new_lang_id)
+	{
+		unsigned int i;
+
+		for (i = 0; i < G_N_ELEMENTS (bom_langs); i++)
+		{
+			if (strcmp (new_lang_id, bom_langs[i]) == 0)
+			{
+				is_bom_lang = TRUE;
+				break;
+			}
+		}
+	}
```

Comment:
I just discovered [g_strv_contains ()](https://developer.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-strv-contains). Maybe it can be used here? This would lead to easier readable code.

Reply:
done